server/webapp/WEB-INF/rails/app/controllers/admin/status_reports_controller.rb

```diff
@@ -47,7 +47,23 @@ def agent_status
       render_plugin_error e
     end
 
+    def cluster_status
```

Comment:
How about adding some specs for this controller method?

Reply:
Done